### ISSUE TITLE
Cann't focus input with the label if the input is not editable

### ISSUE THREAD

**Issue by mickaelw** [halilb/react-native-textinput-effects#38]
I notice if I use the props editable={false} of TextInput to put it in read only mode. But I can focus the field if I click on the label because in the code there is:
`<TouchableWithoutFeedback onPress={this.focus}> `

So I fix the bug for the moment like this in the BaseInput class:
  ```
focus() {
      if(!this.props.editable)
          return
    this.inputRef().focus();
  }
```

**Comment by halilb** [comment]
Thanks for reporting @mickaelw.

Are you using the latest version of the library? This problem should be fixed in 0.3.1, using the same solution you've proposed. You can checkout #30 for more info.

Closing the issue but you're welcome to re-open if there's any other problem I'm missing.

**Comment by mickaelw** [comment]
OK thank you, I update the module soon
